include/aikido/util/CatkinResourceRetriever.hpp

```diff
@@ -1,9 +1,9 @@
 #ifndef AIKIDO_UTIL_CATKINRESOURCERETRIEVER_HPP_
 #define AIKIDO_UTIL_CATKINRESOURCERETRIEVER_HPP_
 #include <string>
-#include <unordered_map>
 #include <vector>
 #include <dart/common/ResourceRetriever.hpp>
+#include <unordered_map>
```

Comment:
Nit: I generally like grouping `#include`s into three blocks: (1) standard library, (2) dependencies, (3) includes relative to this project. I find that this helps identify what binary dependencies (e.g. in this case: DART) a particular file has.

Is this something that we can easily do with `clang-format`? Thoughts?